src/Compilers/CSharp/Portable/Symbols/Tuples/TupleFieldSymbol.cs

```diff
@@ -116,12 +112,12 @@ internal override DiagnosticInfo GetUseSiteDiagnostic()
             return result;
         }
 
-        public override sealed int GetHashCode()
+        public override int GetHashCode()
         {
             return Hash.Combine(_containingTuple.GetHashCode(), _tupleElementIndex.GetHashCode());
         }
```

Comment:
Consider making this an internal helper method, `GetHashCodeInternal`, and change `GetHashCode` to `abstract override` so derived types are forced to override `GetHashCode`. Same comment for `Equals.